src/main/scala/org/broadinstitute/hail/variant/VariantSampleMatrix.scala

```diff
@@ -219,6 +219,23 @@ case class VSMSubgen[T](
   tGen: (Variant) => Gen[T]) {
 
   def gen(sc: SparkContext)(implicit tct: ClassTag[T]): Gen[VariantSampleMatrix[T]] =
+    this.distributeSize.flatMap(x => x.genNoResize(sc))
```

Comment:
My scala-fu is weak. Apparently there are a number of ways to write this expression. Including:

```scala
this.distributeSize flatMap (x => x.genNoResize(sc))
```

```scala
this.distributeSize flatMap (_.genNoResize(sc))
```

is there a preferred style for the hail project?